src/app/find/mapPage/MapPage.jsx

```diff
@@ -220,7 +220,7 @@ export default class MapPage extends Component {
             {this.getCurrentFilterString()}
           </span>
         </div>
-      )}
+        )}
```

Comment:
Odd that this change alone remained, but whatever. :)

Reply:
fixed!